new/PRS13947Eleskndros.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13947Eleskndros" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Ǝleskǝndros</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="VP"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="VP" when="2022-11-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
+                  <persName xml:lang="gez" >እለ፡ እስክንድሮስ</persName>
+                  <persName xml:lang="gez" xml:id="n2">እለ፡ እስክንድሮስ</persName>
+                  <persName xml:lang="gez" type="normalized" corresp="#n2">ʾƎla ʾǝskǝndros</persName>
+                  <persName xml:lang="en" corresp="#n1" sameAs="#n2">Alexander</persName>
+                    <faith type="Traditional">pagan</faith>
+                    <occupation type="ruler">Ruler during the Roman Empire</occupation>
```

Comment:
is he a historical person, also known from other traditions? 
In that case it should be either "Alexander" as `n1 `or at least
```
<persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
       <persName xml:lang="en" corresp="#n1" type="main">Alexander</persName>
```

delete `<persName xml:lang="en" corresp="#n1" sameAs="#n2">Alexander</persName>` this line makes no sense, `sameAs `should be used for wikidata ID (or other authority list)  inside of the `person `element
wikidata ID should be either found on wikidata if he is present, or an ID should be preferably created on wikidata
see https://betamasaheft.eu/Guidelines/?q=sameas&id=persons Example 4

Reply:
Thanks Zhenia. 

It is not known who this Alexander is,  he simply mentioned in the Gadl (also in the Greeck Passio) as  ruler  during the period of the Big Persecutions (reign of  Diocletian), i.e. 303-305, in the region of Cilicia (Asia Minor).  His historicity is unknown.
For the moment I cannot add information about this Alexander. 
I will correct.

Actually, here I wanted to indicate that there are two attested variants of the Geez name, ʾƎleskǝndros and ʾƎla ʾǝskǝndros, both corresponding to the English Alexander. How I did is the right way?

Reply:
Yes but you need the English form as the translation to `n1 `with type `main`, as he is not Ethiopian, see the correction above

Reply:
Thanks.
I did as follows:

  <persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
                    <persName xml:lang="en" corresp="#n1" type="main">Alexander</persName>
                    <persName xml:lang="gez" xml:id="n2">እለ፡ እስክንድሮስ</persName>
                    <persName xml:lang="gez" type="normalized" corresp="#n2">ʾƎla ʾǝskǝndros</persName>
                    <persName xml:lang="en" corresp="#n2" type="main">Alexander</persName>